view/frontend/web/template/payment/cc-form.html

```diff
@@ -60,12 +60,12 @@
                             value: creditCardOwnerBirthMonth,
                             valueUpdate: 'keyup'"
                                     />
-               <input type="text" name="payment[ps_cc_owner_birthday_year]" class="input-text ps_cc_birthdate" value=""
+               <input type="text" name="payment[ps_cc_owner_birthday_year]" class="input-text ps_cc_birthdate_year" value=""
                    data-bind="attr: {
                                     autocomplete: off,
-                                    placeholder: 'YYYY',
+                                    placeholder: 'AAAA',
```

Comment:
It was expected to use the $t() translator, same as used before in the other methods.